ArchUnitNET/Fluent/Extensions/InstructionExtensions.cs

```diff
@@ -42,30 +54,51 @@ public static FieldDefinition GetAssigneeFieldDefinition([NotNull] this Instruct
 
         public static bool IsMethodCallToSetBackingFieldDefinition([NotNull] this Instruction instruction)
         {
-            if (instruction == null) throw new ArgumentNullException(nameof(instruction));
+            if (instruction == null)
+            {
+                throw new ArgumentNullException(nameof(instruction));
+            }
+
             var nextOp = instruction.Next;
-            if (nextOp == null) return false;
+            if (nextOp == null)
+            {
+                return false;
+            }
+
             var assigneeIsBackingField = nextOp.IsOperandBackingField();
-            var isValidSetBakingFieldOp = instruction.IsMethodCallOp() && nextOp.IsSetFieldOp() && assigneeIsBackingField;
+            var isValidSetBakingFieldOp =
+                instruction.IsMethodCallOp() && nextOp.IsSetFieldOp() && assigneeIsBackingField;
             return isValidSetBakingFieldOp;
         }
 
         public static bool IsNewObjectToSetBackingField([NotNull] this Instruction instruction)
         {
-            if (instruction == null) throw new ArgumentNullException(nameof(instruction));
+            if (instruction == null)
```

Comment:
maybe https://github.com/Fody/NullGuard will be helpful

Reply:
In C# 8 all reference types are assumed to be not nullable, so upgrading to C# 8 should make these lines unneccessary anyways.